examples/step-51/doc/intro.dox

```diff
@@ -55,11 +54,22 @@ element-level solutions.  This is known as the
 solution process.
 </ol>
 
-The above procedure also has a linear algebra interpretation and is referred to
-as static condensation. Let us write the complete linear system associated to
-the HDG problem as a block system with the discrete DG (cell interior)
-variables $U$ as first block and the skeleton (face) variables $\Lambda$ as the
-second block:
+<h4> Relation with Static Condensation </h4>
+The above procedure also has a linear algebra interpretation---referred to
+as <i>static condensation</i>---that was exploited to reduce the size of the
+ global linear system by Guyan in the context of continuous Finite Elements
+ @cite G65, and by Fraeijs de Veubeke for mixed methods @cite F65. In the latter
```

Comment:
Oh, these are *old*!!

Reply:
> Oh, these are _old_!!
Just a little bit. The paper by Guyan is fascinating: it is hardly more than half a page long, but the idea is crystal clear. It seems almost unthinkable now to be able to introduce such a remarkable idea in so few lines.